src/components/switch/switch.test.ts

```diff
@@ -21,6 +21,7 @@ describe('<sl-switch>', () => {
     expect(el.required).to.be.false;
     expect(el.checked).to.be.false;
     expect(el.defaultChecked).to.be.false;
+    expect(el.helpText).to.equal('');
```

Comment:
This is the only test I saw in Input and Textarea related to help text. Happy to add more!

Reply:
I think this is fine.